projects/storefrontapp-e2e-cypress/cypress/helpers/cart.ts

```diff
@@ -220,97 +239,45 @@ export function verifyMergedCartWhenLoggedIn() {
 
   cy.get('cx-breadcrumb h1').should('contain', '1 result');
 
-  const miniCart = cy.get('cx-mini-cart');
-  miniCart.within(() => {
-    cy.get('.count').should('contain', 2);
-  });
-  miniCart.click({ force: true });
+  checkMiniCartCount(2).click({ force: true });
 
   cy.get('cx-breadcrumb h1').should('contain', 'Your Shopping Cart');
 
-  getCartItem(product0.name).within(() => {
-    cy.get('.cx-counter-value').should('have.value', '1');
-    cy.get('.cx-total>.cx-value').should('contain', product0.price);
-  });
-
-  getCartItem(product1.name).within(() => {
-    cy.get('.cx-counter-value').should('have.value', '1');
-    cy.get('.cx-total>.cx-value').should('contain', product1.price);
-  });
+  checkProductInCart(product0);
+  checkProductInCart(product1);
 }
 
 export function logOutAndEmptyCart() {
   cy.selectUserMenuOption({
     option: 'Sign Out',
   });
   cy.visit('/cart');
-  cy.get('cx-breadcrumb h1').should('contain', 'Your Shopping Cart');
-  cy.get('.EmptyCartMiddleContent').should(
-    'contain',
-    'Your shopping cart is empty'
-  );
+  validateEmptyCart();
 }
 
 export function manipulateCartQuantity() {
   const product = products[1];
 
   cy.visit(`/product/${product.code}`);
-  cy.get('cx-add-to-cart')
-    .getByText(/Add To Cart/i)
-    .click();
-  cy.get('cx-added-to-cart-dialog .cx-dialog-total').should(
-    'contain',
-    'Cart total (1 item)'
-  );
-  cy.get('cx-added-to-cart-dialog [aria-label="Close"]').click({ force: true });
-
-  const miniCart = cy.get('cx-mini-cart');
-  miniCart.within(() => {
-    cy.get('.count').should('contain', 1);
-  });
-  miniCart.click({ force: true });
+  addToCart();
```

Comment:
Reviewer position doesn't matter, if they request to change superior solution to something other.

Data, good arguments should be the deciding factor for a solution, not a position such as a tech lead.

Reply:
Inform the author of the code if the change you ask for it necessary for approval or it's just nice to have